tools/debian/rules

```diff
@@ -43,19 +43,27 @@ override_dh_auto_configure: debian/git-patches-applied
 		--with-pamdir=/lib/$(DEB_HOST_MULTIARCH)/security \
 		--libexecdir=/usr/lib/cockpit $(CONFIG_OPTIONS)
 
-override_dh_auto_install:
-	dh_auto_install
-	mkdir -p debian/tmp/etc/pam.d
-	install -p -m 644 tools/cockpit.debian.pam debian/tmp/etc/pam.d/cockpit
-
 override_dh_systemd_enable:
 	dh_systemd_enable -p cockpit-ws --name=cockpit cockpit.socket
 
 override_dh_systemd_start:
 	dh_systemd_start -p cockpit-ws cockpit.socket
 
 override_dh_install:
-	dh_install --list-missing -Xusr/src/debug
+	# Debian/Ubuntu PAM config
+	mkdir -p debian/tmp/etc/pam.d
+	install -p -m 644 tools/cockpit.debian.pam debian/tmp/etc/pam.d/cockpit
+
+	# don't ship broken branding symlinks
+	for d in rhel fedora centos; do rm -r debian/tmp/usr/share/cockpit/branding/$$d; done
+	dpkg-vendor --derives-from ubuntu || rm -r debian/tmp/usr/share/cockpit/branding/ubuntu
+
+	# unpackaged modules
```

Comment:
The intent is that we have dangling symlinks for branding we cannot ship. The intent is also that we ship all branding in all packaged versions of Cockpit. The same binaries are often used on other distributions. This happens in the cockpit/ws container.